docs/operator-docs/cumulus-dependencies.md

```diff
@@ -0,0 +1,72 @@
+---
+id: cumulus-dependencies
+title: Cumulus Dependencies
+hide_title: true
+---
+
+# Updating Cumulus Dependencies
+
+
+## Cumulus Dependency Management - A Quick Intro to package.json
+
+As is standard with nodejs projects, Cumulus has lists of dependencies in `package.json` files. For this example, we'll look at the `package.json` file under the `example` directory in the main [Cumulus project](github.com/nasa/cumulus).
```

Comment:
For this link I am getting `Cannot GET /cumulus/docs/next/operator-docs/github.com/nasa/cumulus`